shared/src/proxies/users/getJudgesChambersProxy.ts

```diff
@@ -0,0 +1,18 @@
+import { get } from '../requests';
+
+export interface JudgeChambersInfo {
+  label: string;
+  judgeFullName: string;
+  phoneNumber?: string;
+  section: string;
+  isLegacy?: boolean;
+}
+
+export const getJudgesChambersInteractor = (
```

Comment:
Optional: If you want you can use the original getUserBy section endpoint and do all the formatting inside of your action instead of making a new endpoint

Reply:
Done! (Locally! Will push soon.)